public/src/components/channelManagement/epicTests/epicVariantPreview.tsx

```diff
@@ -188,7 +188,7 @@ const buildProps = (variant: EpicVariant): EpicProps => ({
 
 const useStyles = makeStyles(({}: Theme) => ({
   container: {
-    maxWidth: '620px',
+    width: '620px',
```

Comment:
Question: what would happen if we set a maxwidth here to the mobile breakpoint value? Would the epics display with mobile copy/CTAs? If yes, then maybe we could add a button to switch between mobile/other widths?

Reply:
@KaliedaRik just linked me to this feature - https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries
I will experiment